product/server.js

```diff
@@ -0,0 +1,7 @@
+import app from './src/app.js'
+
+const port = process.env.PORT || 3000;
```

Comment:
are you using .env?

Reply:
not yet, Its there because alura's instructor used it, I think we'll use it in future sprints